habinezadalvan/LIG-app-api
# chore-LUX(refactor): refactor sendMail function

### Description
- create main sendEmail html
- tests

### How to test

- clone repo

- checkout to chore-LUX-code-refactoring

- check directory /src/utils/mailer

### How has this been tested?


### Jira
none
